upgrade/upgrade.go

```diff
@@ -231,6 +232,12 @@ func doResourceUpgrade(namespace string, lhClient *lhclientset.Clientset, kubeCl
 			return err
 		}
 	}
+	if semver.Compare(lhVersionBeforeUpgrade, "v1.5.2") < 0 {
+		logrus.Info("Walking through the resource upgrade path v1.5.1 to v1.5.2")
```

Comment:
Nit: this can be the case that we are upgrading from v1.4.x/v1.5.0/v1.5.0 to v1.5.2 right?

Reply:
You are correct. If your concern is just the way the log message is worded, I chose to stick to the convention we have previously followed.

https://github.com/longhorn/longhorn-manager/blob/47d1057f6e0c15f337af691bd91d54d0ce849b14/upgrade/upgrade.go#L281-L292 

Perhaps it would be better if we just dropped the first number in these messages? (E.g. `Walking through the resource upgrade path for v1.5.2".)

Reply:
Agreed, but perhaps "... to v1.5.2".  That removes the slight ambiguity of "for".

Reply:
> Agreed, but perhaps "... to v1.5.2". That removes the slight ambiguity of "for".

This approach sounds good to me